lib/favicons.js

```diff
@@ -54,11 +55,13 @@ function generateIcons (loader, imageFileStream, pathPrefix, query, callback) {
     appName: query.appName
   }, function (err, result) {
     if (err) return callback(err);
-    var html = result.html.filter(function (entry) {
-      return entry.indexOf('manifest') === -1;
```

Comment:
@jantimon can you unpack this a little? What do you mean that the quality is bad? Are you saying that the manifest spec is poor? Or that the implementation is bad? What can we do as contributors to this project to improve matters?

If you feel strongly that it shouldn't be part of the implementation, would it be an acceptable compromise to make the default option to omit it, but allow users to opt in if they're happy to put up with whatever's wrong?